src/Docker.DotNet/Endpoints/IPluginOperations.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Docker.DotNet.Models;
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Docker.DotNet
+{
+    public interface IPluginOperations
+    {
+        /// <summary>
+        /// List plugins.
+        /// </summary>
+        /// <remarks>
+        /// docker plugin ls
+        ///
+        /// HTTP GET /plugins
+        ///
+        /// 200 - No error.
+        /// 500 - Server error.
+        /// </remarks>
+        Task<IList<PluginListResponse>> ListPluginsAsync(PluginListParameters parameters, CancellationToken cancellationToken = default(CancellationToken));
+        
```

Comment:
Mind adding an entry for `GetPluginPrivilegesAsync` and having it throw `NotImplementedException()`? Will make it super easy to see whats 'up-for-grabs'

Reply:
Also your generated `Plugin` but then you copied the type to `PluginListResponse`. Dont you want `IList<Plugin>` here?